src/app/pages/FrontPage/FrontPage.jsx

```diff
@@ -76,13 +69,6 @@ const FrontPage = ({ pageData, mostReadEndpointOverride }) => {
 
   const { isAmp, showAdsBasedOnLocation } = useContext(RequestContext);
 
-  const offScreenText = (
-    // eslint-disable-next-line jsx-a11y/aria-role
-    <span role="text">
-      <span lang="en-GB">{product}</span>, {serviceLocalizedName} - {home}
-    </span>
-  );
-
```

Comment:
should this have been removed @greenc05

Reply:
No, this isn't anything to do with this PR unless I'm mistaken

Reply:
@mukama sorry we need to add this back in so that the AT has something to read out